webapp/channels/src/components/resizable_sidebar/resizable_divider.tsx

```diff
@@ -100,7 +100,7 @@ function ResizableDivider({
     const currentUserID = useSelector(getCurrentUserId);
 
     const [isActive, setIsActive] = useState(false);
-    const [width, setWidth] = useGlobalState<number | null>(null, `resizable_${name}:${currentUserSuffix}`, currentUserID);
```

Comment:
this was a selector function, not a string, so incompatible as-used; removed as suffixed by 3rd param/currentUserID